src/components/intro/index.tsx

```diff
@@ -113,7 +113,12 @@ export default class Intro extends Component<Props, State> {
     this.setState({ beforeInstallEvent: event });
 
     // Log the event.
-    ga('send', 'event', 'pwa-install', 'available');
+    const gaEventInfo = {
+      eventCategory: 'pwa-install',
+      eventAction: 'promo-shown',
+      nonInteraction: true,
+    };
+    ga('send', 'event', gaEventInfo);
```

Comment:
Big change here is that I've set `nonInteraction`, this doesn't have a big effect, but if it's not set, the bounce rates are incorrectly calculated, because Analytics *thinks* the user did something, but they haven't.